File: Aulas/aula05/calculadora.js
// calculadora para operações básicas com dois números.


// Pedir a entrada do usurio
const input = require("prompt-sync")();

console.log("Calculadora básica");

console.log(`
    Digite:
    1 - Somar
    2 - Subtrair
    3 - Multiplicar
    4 - Dividir
    0 - Sair
    `)
operacao = Number(input('Escolha uma operacao:  '));
numero1 = Number(input("Digite o primeiro número:  "));
numero2 = Number(input("Digite o segundo número:  "));


switch (operacao) {
    case 1: 
        console.log(numero1 + numero2);
        break;
    case 2: 
        console.log(numero1 - numero2);
        break;
    case 3: 
        console.log(numero1 * numero2);
        break;
    case 4: 
        if (numero2 ==  0) {
            console.log("Error zero division");
        }
        console.log(numero1 / numero2);
        break;
    case 0: 
    
        break;
    default:
        console.log("Operação inválida!");

}

File: Aulas/aula61/aula61Java/src/Calculadora.java
import java.util.Scanner;

public class Calculadora {
    public static void main(String[] args) {
        Scanner input = new Scanner(System.in);
        System.out.println("Calculadora");
        System.out.println("Escolha a operação:\n1-Somar\n2-Subtrair\n3-Multiplicar\n4-Dividir\n\n");
        int operation = input.nextInt();

        System.out.println("Digite o primeiro número\n");
        int number1 = input.nextInt();

        System.out.println("Digite o segundo número\n");
        int number2 = input.nextInt();


        switch (operation) {
            case 1:
                System.out.println("O Resultado é: " + (number1 + number2));
                break;

            case 2:
                System.out.println("O Resultado é: " + (number1 - number2));
                break;

            case 3:
                System.out.println("O Resultado é: " + (number1 * number2));
                break;

            case 4:
                if(number2 != 0) {
                    System.out.println("O Resultado é: " + (number1 / number2));
                } else {
                    System.out.println("Erro Divisão por Zero!");
                }

                break;

            default:
                System.out.println("Operação inválida!");
        }
    }
}
